cognitive/src/main/scala/com/microsoft/ml/spark/cognitive/TextAnalyticsSDK.scala

```diff
@@ -181,3 +183,24 @@ class TextSentimentV4(override val textAnalyticsOptions: Option[TextAnalyticsReq
   }
 
 }
+
+object PII extends ComplexParamsReadable[PII]
```

Comment:
[Not for this PR] - I'm not sure what the right naming convention, but I think we should rename our objects a bit to include nouns. @mhamilton723 any thoughts?

Reply:
will correct it